astropy/io/ascii/ui.py

```diff
@@ -229,18 +244,6 @@ def _guess(table, read_kwargs, format, fast_reader):
     call make sure that if there is a corresponding definition in the guess
     then it must have the same val.  If not then skip this guess."""
```

Comment:
Maybe the point has come to put docs for parameters and return values on here, even for an internal function, because it reaches a level of complexety where some help might be needed for anybody who wants to maintain it except the few people to follow this module really closely.